packages/suite/src/components/suite/SecurityCheck/DeviceCompromised.tsx

```diff
@@ -48,12 +49,22 @@ export const DeviceCompromised = () => {
         if (hashCheckError) reportCheckFail('hash', { ...commonData, hashCheckError });
     }, [revisionCheckError, hashCheckError, revision, vendor, version]);
 
+    const isSoftFailure = revisionCheckError === null && hashCheckError === 'other-error';
```

Comment:
A countefeit device could deliberately crash hash check plus fail revision check, so if revision check did not pass, it has precedence (:arrow_right: display normal ghost modal)